src/atopile/cli/create.py

```diff
@@ -12,7 +14,9 @@
 import git
 import jinja2
 import rich
-import rich.prompt
+import yaml
```

Comment:
If you switch this out for `ruamel.yaml` we'll preserve people's YAMLs

eg. 
```python
import ruamel.yaml

yaml = ruamel.yaml.YAML()  # defaults to round-trip
```

See: https://yaml.readthedocs.io/en/latest/detail/#addingreplacing-comments